src/twisted/python/compat.py

```diff
@@ -342,7 +342,7 @@ def ioType(fileIshObject, default=str):
     return default
 
 
-def nativeString(s):
+def nativeString(s: AnyStr) -> str:
```

Comment:
Since you are here, you might want to change the docstring to L{bytes}, L{str}.

Reply:
a'ight